src/android/com/megster/cordova/BluetoothSerialService.kt

```diff
@@ -49,7 +50,9 @@ object BluetoothSerialService {
             mState = state
 
             if (mState == STATE_CONNECTED) {
-                connectedCallback?.connected()
+                connectedCallback?.connected(remoteDeviceMacAddress)
+            } else {
+                remoteDeviceMacAddress = null
```

Comment:
anytime the state is changed to anything but connected the remote device mac address should be cleared out, it will get reset when connect completes